test/util/test_runners/test_runner_args.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+
+
+class TestRunnerArgs:
+    """
+    DTO to hold the tests environment arguments
+    """
+
+    def __init__(self, hostname, account, username, password, invoke_cli_as_process,
+                 working_cli_path_only_for_init: str,
+                 cli_to_test_path: str, files_folder: str, test_name_identifier="integration"):
+        self.hostname = hostname
+        self.account = account
+        self.username = username
+        self.password = password
+        self.test_name_identifier = test_name_identifier
+        self.invoke_cli_as_process = invoke_cli_as_process
+        self.working_cli_path_only_for_init = working_cli_path_only_for_init
+        self.cli_to_test_path = cli_to_test_path
+        self.files_folder = files_folder
+        # will throw if the user test the same known working binaries
+        if cli_to_test_path is not None and working_cli_path_only_for_init.lower() == cli_to_test_path.lower():
```

Comment:
what do you mean if a user tests the same known working binaries? As in if there are two cli executables?

Reply:
working_cli_path_only_for_init is a known working binaries meant to run the init and login command to set up the test environment.
cli_to_test_path is the cli to test.
if they both have the same path we warn the user. it can be override by the use of -s parameter